scripts/test_travis.sh

```diff
@@ -28,6 +28,7 @@ case "$1" in
     ;;
 
     integration)
+        ./scripts/build_aether_containers.sh
```

Comment:
maybe `./scripts/build_aether_utils_and_distribute.sh` ? Because you want it out of `test_integration.sh` file :thinking: